include/sot/torque_control/free-flyer-locator.hh

```diff
@@ -79,8 +80,12 @@ namespace dynamicgraph {
         /* --- CONSTRUCTOR ---- */
         FreeFlyerLocator( const std::string & name );
 
-        void init(const std::string& urdfFile);
+        void init(const std::string& urdfFile,
+		  const std::string & leftFootFrameName,
+		  const std::string & rightFootFrameName);
 
+	/// Set the mapping between urdf and sot.
+	void setJoints(const dynamicgraph::Vector &);
```

Comment:
It is not obvious to translate the mapping in something easily understandable by the SoT type caster. Could you have a look to test_control_manager.py:33 for an alternative and give some feedback ? Thanks for your help.